docs/repo-specific-configuration.md

```diff
@@ -32,6 +32,41 @@ You can add `<YOUR_REPO>/.scala-steward.conf` to configure how Scala Steward upd
 #pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
 pullRequests.frequency = "7 days"
 
+# pullRequests.grouping allows you to specify how Scala Steward should group
+# your updates in order to reduce the number of pull-requests.
+#
+# Updates will be placed in the first group with which they match, starting
+# from the first in the array. Those that do not match any group will follow
+# the default procedure (one PR per update).
+#
+# Each element in the array will have the following schema:
+# 
+#   - name (mandatory): the name of the group, will be used for things like naming the branch
+#   - title (optional): if provided it will be used as the title for the PR
+#   - filter (mandatory): a non-empty list containing the filters to use to know 
+#                         if an update falls into this group.
+#
+# `filter` properties would have this format:
+#   
+#    {
+#       version = "major" | "minor" | "patch" | "pre-release" | "build-metadata",
```

Comment:
@alejandrohdezma I think it is better to explain the differences of these `version` enums.
Specially, I do not get `build-metadata` 🙇

Reply:
Maybe it can be a good idea to refer to the [semver.org](https://semver.org) docs on these docs. WDYT?

On `build-metadata` from point [10](https://semver.org/#spec-item-10):

> Build metadata MAY be denoted by appending a plus sign and a series of dot separated identifiers immediately following the patch or pre-release version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85, 1.0.0+21AF26D3—-117B344092BD.

Reply:
Ah it is server terms.
Yeah, a link to that sounds enough